src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -617,7 +552,6 @@ impl<S: S3BackEnd + Sync + Send + 'static> MetaData for S3MetaData<S> {
             if let SFlag::S_IFREG = node.get_type() {
                 let ino = node.get_ino();
                 self.storage.remove(ino).await;
-                self.local_mtime_and_size.remove(&ino);
```

Comment:
We don't maintain `open_files` in delete related functions, because the open file entry must be removed when `release()` called, right?

Reply:
`lookup_cnt == 0` means no kernel still use the node. In other words, the file is not open.